web/modules/custom/collection/collection.module

```diff
@@ -144,15 +143,15 @@ function collection_entity_bundle_field_info_alter(&$fields, EntityTypeInterface
  * Implements hook_entity_insert().
  */
 function collection_entity_insert(EntityInterface $entity) {
-  if ($entity->bundle() !== 'collection') {
+  if ($entity->getEntityTypeId() !== 'rdf_entity' || $entity->bundle() !== 'collection') {
     return;
   }
   collection_entity_update($entity);
 
   // When a new collection is proposed, give the user the 'group owner' role.
   // This will not occur if the user is the anonymous, i.e. the entity was
   // created by some batch process or some importing.
-  if (Og::isGroup($entity->getEntityTypeId(), $entity->bundle()) && !\Drupal::currentUser()->isAnonymous()) {
+  if (Og::isGroup($entity->getEntityTypeId(), $entity->bundle()) && !empty($entity->getOwner())) {
```

Comment:
Ok the membership is not found (a few lines later) when the user is anonymous, so the code bails out. It works.